io.github.materialgram.yml

```diff
@@ -0,0 +1,553 @@
+      - -DJPEGXL_ENABLE_OPENEXR=OFF
+      - -DJPEGXL_ENABLE_SKCMS=OFF
+    sources:
+      - type: archive
+        url: https://github.com/libjxl/libjxl/archive/v0.8.2/libjxl-0.8.2.tar.gz
+        sha256: c70916fb3ed43784eb840f82f05d390053a558e2da106e40863919238fa7b420
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/libjxl/libjxl/releases/latest
+          tag-query: .tag_name
+          timestamp-query: .published_at
+          version-query: $tag | sub("^[vV]"; "")
+          url-query: '"https://github.com/libjxl/libjxl/archive/\($tag)/libjxl-\($version).tar.gz"'
+
+  - name: libsigc++
+    buildsystem: meson
+    builddir: true
+    config-opts:
+      - --buildtype=plain
+      - -Dbuild-examples=false
+      - -Dvalidation=false
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/libsigc++/3.6/libsigc++-3.6.0.tar.xz
+        sha256: c3d23b37dfd6e39f2e09f091b77b1541fbfa17c4f0b6bf5c89baef7229080e17
+        x-checker-data:
+          type: gnome
+          name: libsigc++
+
+  - name: glib
+    buildsystem: meson
+    builddir: true
+    config-opts:
+      - --buildtype=plain
+      - -Dtests=false
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/glib/2.78/glib-2.78.1.tar.xz
+        sha256: 915bc3d0f8507d650ead3832e2f8fb670fce59aac4d7754a7dab6f1e6fed78b2
+        x-checker-data:
+          type: gnome
+          name: glib
+    cleanup:
+      - /bin
+
+  - name: glibmm
+    buildsystem: meson
+    builddir: true
+    config-opts:
+      - --buildtype=plain
+      - -Dbuild-examples=false
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/glibmm/2.78/glibmm-2.78.0.tar.xz
+        sha256: 5d2e872564996f02a06d8bbac3677e7c394af8b00dd1526aebd47af842a3ef50
+        x-checker-data:
+          type: gnome
+          name: glibmm
+
+  - name: mozjpeg
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=None
+      - -DCMAKE_INSTALL_DEFAULT_LIBDIR=lib
+      - -DENABLE_STATIC:BOOL=OFF
+    sources:
+      - type: archive
+        url: https://github.com/mozilla/mozjpeg/archive/v4.1.1/mozjpeg-4.1.1.tar.gz
+        sha256: 66b1b8d6b55d263f35f27f55acaaa3234df2a401232de99b6d099e2bb0a9d196
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/mozilla/mozjpeg/releases/latest
+          tag-query: .tag_name
+          timestamp-query: .published_at
+          version-query: $tag | sub("^[vV]"; "")
+          url-query: '"https://github.com/mozilla/mozjpeg/archive/\($tag)/mozjpeg-\($version).tar.gz"'
+    cleanup:
+      - /bin
+
+  - name: rnnoise
+    config-opts:
+      - --disable-static
+      - --disable-examples
+      - --disable-doc
+    sources:
+      - type: git
+        url: https://gitlab.xiph.org/xiph/rnnoise.git
+        commit: 7f449bf8bd3b933891d12c30112268c4090e4d59
+
+  - name: qtbase
```

Comment:
you should use the KDE runtime then

Reply:
do I need to add only kde runtime or I need to have both kde runtime and freedesktop runtime?